### DISCUSSION TITLE
If original file is WebP, convert it to JPG

### DISCUSSION THREAD

**Discussion by ainsleyclark** [payloadcms/payload#8804] [Q&A] [answered]
Hey,

If a file is uploaded to Payload which is `image/webp`, is there a way to convert it to a JPG beforehand? I.e I don't want to store webps as the original format for fallbacks in the browser.

**Comment by akhrarovsaid** [comment, marked as answer]
Hey @ainsleyclark,

I think what you can do is pass a `formatOptions` object to your `Upload` collection. [See docs](https://payloadcms.com/docs/beta/upload/overview#collection-upload-options)

Maybe something like:
```
upload: {
    formatOptions: {
      format: 'jpeg',
    },
},
```

**Reply by ainsleyclark** [reply]
Amazing @akhrarovsaid! Thanks
